### PR TITLE
Upgrade cardano-node executable to v8.4.0-pre

### PR DESCRIPTION
**Pull request by koslambrou**
Pre-submit checklist:
- Branch
    - [ ] Tests are provided (if possible)
    - [ ] Commit sequence broadly makes sense and have useful messages
    - [ ] Important changes are reflected in changelog.d of the affected packages
    - [ ] Relevant tickets are mentioned in commit messages
- PR
    - [ ] (For external contributions) Corresponding issue exists and is linked in the description
    - [ ] Targeting main unless this is a cherry-pick backport
    - [ ] Self-reviewed the diff
    - [ ] Useful pull request description
    - [ ] If relevant, reference the ADR in the PR and reference the PR in the ADR
    - [ ] Reviewer requested

### CODE REVIEW

File: nix/haskell.nix [hunk 1]
```diff
@@ -1,16 +0,0 @@
-# This file is part of the IOGX template and is documented at the link below:
```
Comment (koslambrou) [inline nix/haskell.nix:1]: @zeme-wana Why did you remove this? Seems like it's still part of IOGX

Comment (zeme-wana) [reply nix/haskell.nix:1]: Yes but the link is obsolete. 
Soon I'll be updating all the readmes in the repos using iogx so that they have up-to-date docs.
